src/include/daos/common.h

```diff
@@ -149,15 +149,27 @@ char *DP_UUID(const void *uuid);
 #define DF_CONTF		DF_UUIDF"/"DF_UUIDF
 
 #ifdef DAOS_BUILD_RELEASE
-#define DF_KEY			"[%d]"
-#define DP_KEY(key)		(int)((key)->iov_len)
+
+#define DF_KEY       "key[%d]"
```

Comment:
this change is not in the PR description. can you explain why you are changing the logging for DF_KEY?
i'd rather not.

Reply:
I thought I'd replied to this earlier but can't see it now, the details are all in https://daosio.atlassian.net/wiki/spaces/~60357e57e2020c0070b4d71f/pages/11403198500/Keeping+information+out+of+log+files.

Reply:
this does not address my question, although i was not aware of this proposal. it might be a good topic to discuss at the PDT. also i will mention again that the pr title and description don't mention anything about ppi cleanup and is only about dentries. so this PPI cleanup for keys should be in a different PR.

aside from that, for this change you are adding this "key" logging to DF_KEY. this is not going to be repetitive in most logs i would assume, as the log message can have dkey or akey followed by DF_KEY. this is my requested change for this PR.

Reply:
I didn't think that three additional characters in some log lines was a big enough change in itself to warrant an individual ticket/PR for.

A part of this overall change is having different types of strings in the log file represented in a consistent way, so updating the existing macros (of which we currently only have keys) at the same time as adding a second pair of macros made sense to me.

I can update the description to include this detail.

Reply:
well im not sure adding the type is a good idea. as i mentioned for keys, the type can be a dkey or akey and the log message usually have that already/

Reply:
Looking at the existing use of `DF_KEY` in the source you're right, a lot of it already does include an `akey` or `dkey` prefix already but it's very inconsistent with the exact syntax.  There are only ~100 uses of DF_KEY in the source which is much less than I thought, will revert this part of the PR and handle this afterwards.